scripts/zones/Dynamis-Xarcabard/mobs/Dynamis_Lord.lua

```diff
@@ -52,14 +52,21 @@ function onMobFight(mob,target)
     local YingID = 17330183;
     local YangID = 17330184;
 
-    if (mob:getBattleTime() % 90 == 0) then
-        if (GetMobAction(YingID) == ACTION_NONE) then
-            SpawnMob(YingID):updateEnmity(target); -- Respawn Ying after 90sec
-        end
-        if (GetMobAction(YangID) == ACTION_NONE) then
-            SpawnMob(YangID):updateEnmity(target); -- Respawn Yang after 90sec
+    if (mob:getBattleTime() % 90 == 0 and mob:getBattleTime() >= 90) then
+        if (GetMobAction(YingID) == ACTION_NONE and GetMobAction(YangID) == ACTION_NONE) then
+            GetMobByID(YingID):setSpawn(-414.282,-44,20.427); -- These come from DL's spawn point when he spawns them.
```

Comment:
That's what I don't know. In any case, they should probably move back to their spawn point instead of where the one dragon spawned the other. I'm going to make the assumption that they wander back to where DL spawns them in the event of a wipe.